src/app/patient-table/patient-table.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core'
+import { PatientService } from './../services/patient-service'
+@Component({
+  selector: 'app-patient-table',
+  templateUrl: './patient-table.component.html',
+  styleUrls: ['./patient-table.component.css'],
+})
+export class PatientTableComponent {
+  dataSource!: any[]
+  constructor(private patientService: PatientService) {}
+  ngOnInit(): void {
+    this.patientService.getPatients().subscribe((patients) => {
+      this.dataSource = patients
```

Comment:
to be able to catch error when mockoon in not connected please use this structure to display error in snackbar

{
            next: (data: Array<Patient>) => (this.dataSource = patients),
            error: (err) => {
              this.snackService.showSnackBarMessage(
                'ERROR.PATIENT_TABLE_GET_PATIENTS',
                SNACK_TYPE.error,
              )
              console.log(err.message)
            },
          }